### DISCUSSION TITLE
bluesky_cli delete user from a (mute) list

### DISCUSSION THREAD

**Discussion by kesma01** [myConsciousness/atproto.dart#954] [Q&A] [answered]
Hi,

i just got introduced and immediately started using it as a powerful tool to manage my mute lists used in feeds.
I just can not delete a member from a mute list. Is this possible? If it is, can a get a hint or two, how this should be done.

Thank you and best regards,
Kesma

**Comment by myConsciousness** [comment, marked as answer]
Hey @kesma01 ,

Since the list members are treated as records of `listitem`, all these data can be deleted with the `delete` command.

So I guess you can delete it like:

```bash
bsky delete --uri=at://foo.com/app.bsky.graph.listitem/123
```

But perhaps you need to remember the AT URI (`listitem`) returned when you add a member to this list.

**Reply by kesma01** [reply]
Thank you a lot @myConsciousness. I will run few experiments based on these insights.
